services/minespace-web/src/styles/components/Tables.scss

```diff
@@ -5,6 +5,11 @@
   overflow-y: auto;
 }
 
+.wrapped-text {
+  word-break: break-word;
```

Comment:
Based on the documentation break-word is deprecated. Should be normal:
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word